rpcs3/Emu/RSX/VK/VKGSRender.cpp

```diff
@@ -2699,7 +2720,8 @@ void VKGSRender::prepare_rtts(rsx::framebuffer_creation_context context)
 	}
 
 	auto vk_depth_format = (layout.zeta_address == 0) ? VK_FORMAT_UNDEFINED : vk::get_compatible_depth_surface_format(m_device->get_formats_support(), layout.depth_format);
-	m_current_renderpass_id = vk::get_render_pass_location(vk::get_compatible_surface_format(layout.color_format).first, vk_depth_format, (u8)m_draw_buffers.size());
+	auto vk_color_format = vk::get_compatible_surface_format(layout.color_format).first;
```

Comment:
why is this necessary?

Reply:
moving this out as a local variable allowed me to use it in the exception text, as well as arguably makes the code more readable

Reply:
makes sense for the variable usage, but I still think a helper function is better suited for that.